### ISSUE TITLE
Staging Review finding: On preferences selection, focus shifts to page heading

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#134538]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Authenticated Experience
**Product:** Profile on va.gov
**Feature:** Profile 2.0

## Findings details

**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)

**Launch-blocking:** No 
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

On Edit contact preferences, when using the keyboard and selecting a contact preference, on selection focus moves to the page heading. 

Other occurences:
- Edit contact preferences radio selection
- Edit contact preferences checkbox selection
- Edit appointment preferences

Tested:
- Chrome w/keyboard
- Chrome/JAWS
- Safari/Voiceover

<details><summary>Link, screenshot or steps to recreate</summary>

https://github.com/user-attachments/assets/8743e888-9b88-4443-b52b-a2ae8d8127b1

</details>

## Recommended action

Ensure focus remains on the select, to maintain context for the user. 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 3.2.2 On Input - Level A](https://www.w3.org/TR/WCAG22/#on-input)
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
